pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "hatchling.build"
 
 [project]
 name = "algorand-python-testing"
-version = "0.2.2-beta.1"
+version = "0.2.2-beta.2"
```

Comment:
I assume the 0.1.0 release would be because of a `feat:` conventional commit, so it would start with `0.0.0` find the `feat` commit and then infer the next version should be `0.1.0`